src/main/java/ui/GUIController.java

```diff
@@ -22,11 +27,29 @@
 
 public class GUIController {
 
+    private static final Logger logger = HTLog.get(GUIController.class);
+
     private final PanelControl panelControl;
     private final UI ui;
     private final Label apiBox;
+
+    /**
+     * The previous amount of the available remaining API requests.
+     */
+    private int previousRemainingApiRequests = 0;
+
+    /**
+     * The previous amount of API calls used.
+     */
+    private int apiCallsUsedInPreviousRefresh = 0;
+
     private String defaultRepoId;
 
+    /**
+     * The duration between each refresh of the data store.
+     */
+    private long refreshDurationInMinutes = 1;
```

Comment:
I wonder if these three variables can be moved to a more suitable place. `ApiQuotaManager` or `ApiQuotaInfo`?